### PR TITLE
Fix dependency for some tests

### PR DESCRIPTION
**Pull request by kaisun2000**
### Issues

- [x] My PR addresses the following Helix issues and references them in the PR description:

#1226 etc

### Description

- [x] Here are some details about my PR, including screenshots of any UI changes:

Fix dependency for some tests. Some of the test running
under surefire + testng would interleave the test cases in
the same test class to other test classes. Examine the log
would be very confusing.

Also enhance the base test case logging a little bit to
output both test class and test case names.

### Tests

- [ ] The following tests are written for this issue:

(List the names of added unit/integration tests)

- [x] The following is the result of the "mvn test" command on the appropriate module:

[ERROR] Tests run: 1251, Failures: 1, Errors: 0, Skipped: 5, Time elapsed: 5,867.616 s <<< FAILURE! - in TestSuite
[ERROR] beforeMethod(org.apache.helix.tools.TestClusterStateVerifier)  Time elapsed: 4.536 s  <<< FAILURE!
org.apache.helix.HelixException: cluster null is not setup yet
	at org.apache.helix.tools.TestClusterStateVerifier.beforeMethod(TestClusterStateVerifier.java:57)

[INFO] 
[INFO] Results:
[INFO] 
[ERROR] Failures: 
[ERROR] org.apache.helix.tools.TestClusterStateVerifier.beforeMethod(org.apache.helix.tools.TestClusterStateVerifier)
[ERROR]   Run 1: TestClusterStateVerifier.beforeMethod:57 » Helix cluster null is not setup yet
[INFO]   Run 2: PASS
[INFO] 
[INFO] 
[ERROR] Tests run: 1249, Failures: 1, Errors: 0, Skipped: 3
[INFO] 
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
### Documentation (Optional)

- In case of new functionality, my PR adds documentation in the following wiki page:

(Link the GitHub wiki you added)

### Commits

- My commits all reference appropriate Apache Helix GitHub issues in their subject lines. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  1. Subject is limited to 50 characters (not including Jira issue reference)
  1. Subject does not end with a period
  1. Subject uses the imperative mood ("add", not "adding")
  1. Body wraps at 72 characters
  1. Body explains "what" and "why", not "how"

### Code Quality

- My diff has been formatted using helix-style.xml 
(helix-style-intellij.xml if IntelliJ IDE is used)

### CODE REVIEW

File: helix-core/src/test/java/org/apache/helix/integration/TestPartitionMovementThrottle.java [hunk 1]
```diff
@@ -278,7 +278,7 @@ public void testThrottleOnlyClusterLevelAnyType() {
     ClusterLiveNodesVerifier liveNodesVerifier =
         new ClusterLiveNodesVerifier(_gZkClient, CLUSTER_NAME,
             Lists.transform(Arrays.asList(_participants), MockParticipantManager::getInstanceName));
-    Assert.assertTrue(liveNodesVerifier.verifyByZkCallback(1000));
+    Assert.assertTrue(liveNodesVerifier.verifyByZkCallback(5000));
```
Comment (jiajunwang) [inline helix-core/src/test/java/org/apache/helix/integration/TestPartitionMovementThrottle.java:281]: TestHelper.WAIT_DURATION?

**Review by jiajunwang — COMMENTED** [review]
Seems the test cannot finish in the github run result. Could you please update the test result in your local environment?

File: helix-core/src/test/java/org/apache/helix/integration/TestPartitionMovementThrottle.java (hunk 1 above)
Comment (kaisun2000) [reply helix-core/src/test/java/org/apache/helix/integration/TestPartitionMovementThrottle.java:281]: good point. fixed.

**Comment by kaisun2000** [comment]
> Seems the test cannot finish in the github run result. Could you please update the test result in your local environment?

The following is the failure. The TestClusterStateVerifier is introduced https://github.com/apache/helix/pull/1534/. Note `_clusterName = className + "_" + methodName;` is missed. The fix is in https://github.com/apache/helix/pull/1541/. 
So this PR is looking good. 

Note, without #1541 in, current master would have consistent failure like this.

>2020-11-18T23:00:01.1037581Z [ERROR] Tests run: 1251, Failures: 1, Errors: 0, Skipped: 5, Time elapsed: 5,185.592 s <<< FAILURE! - in TestSuite
2020-11-18T23:00:01.1039247Z [ERROR] beforeMethod(org.apache.helix.tools.TestClusterStateVerifier)  Time elapsed: 5.452 s  <<< FAILURE!
2020-11-18T23:00:01.1040860Z org.apache.helix.HelixException: cluster null is not setup yet
2020-11-18T23:00:01.1043135Z 	at org.apache.helix.tools.TestClusterStateVerifier.beforeMethod(TestClusterStateVerifier.java:57)
2020-11-18T23:00:01.1044683Z 
2020-11-18T23:00:01.5017540Z [ERROR] Failures: 
2020-11-18T23:00:01.5020256Z [ERROR] org.apache.helix.tools.TestClusterStateVerifier.beforeMethod(org.apache.helix.tools.TestClusterStateVerifier)
2020-11-18T23:00:01.5031613Z [ERROR]   Run 1: TestClusterStateVerifier.beforeMethod:57 » Helix cluster null is not setup yet
2020-11-18T23:00:01.5032997Z [ERROR] Tests run: 1249, Failures: 1, Errors: 0, Skipped: 3

**Review by jiajunwang — APPROVED** [review]
@kaisun2000 Thanks for the follow-up. Could you please rebase to the master and then we can see the test result again?

**Comment by jiajunwang** [comment]
@kaisun2000 Since we have already claimed that the test is now stabilized enough, is this PR still needed?

**Comment by jiajunwang** [comment]
Close due to inactive.

**Pull request closed without merging**